conf/cfc.config

```diff
@@ -39,20 +39,22 @@ params {
  * `-plugins nf-co2footprint@<VERSION>` parameter in your nextflow call.
  */
 co2footprint {
-    // Defines the timestamp model that is used as a suffix of the file
-    String co2_timestamp = params.get('trace_report_suffix') ?: this.hasProperty('trace_timestamp') ? this.getProperty('trace_timestamp') : null
-    if (co2_timestamp != null) {
-        timestamp = co2_timestamp
+    // Determine timestamp suffix
+    def timestampSuffix = params.get('trace_report_suffix') ?: (this.hasProperty('trace_timestamp') ? this.trace_timestamp : '')
```

Comment:
So this mean currently, that if there is no (known) timestamp given already, `''` will be used for the plugin. Shouldn't then a new one be created? 
Not sure how often this occurs, but for old nf-core pipeline versions this is the case.